### PR TITLE
Fix InteractiveWindow package name in insertion tool

### PR DESCRIPTION
**Pull request by tmat**

### CODE REVIEW

**Comment by tmat** [comment]
@balajikris This should fix the interactive window entry in LanguageServices.wsx

File: src/Setup/DevDivInsertionFiles/BuildDevDivInsertionFiles.vb [hunk 1]
```diff
@@ -486,7 +486,7 @@ Public Class BuildDevDivInsertionFiles
         ' TODO: remove
         Public ReadOnly Property IsInteractiveWindow As Boolean
             Get
-                Return PackageName = "Microsoft.VisualStudio.InteractiveWindow"
+                Return PackageName = "Microsoft.VisualStudio.InteractiveWindow" OrElse PackageName = "Microsoft.VisualStudio.VsInteractiveWindow"
             End Get
```
Comment (balajikris) [inline src/Setup/DevDivInsertionFiles/BuildDevDivInsertionFiles.vb]: can it really be of either names?

**Comment by balajikris** [comment]
:+1:

File: src/Setup/DevDivInsertionFiles/BuildDevDivInsertionFiles.vb (hunk 1 above)
Comment (tmat) [inline src/Setup/DevDivInsertionFiles/BuildDevDivInsertionFiles.vb]: Yeah, we have 2 packages. I haven't realized that when adding this.

**Pull request merged**